playwright/__main__.py

```diff
@@ -13,17 +13,24 @@
 # limitations under the License.
 
 import os
+import platform
 import subprocess
 import sys
 
 from playwright._impl._driver import compute_driver_executable
+from playwright._repo_version import version
 
 
 def main() -> None:
     driver_executable = compute_driver_executable()
```

Comment:
please move that also into the else.

Reply:
Conditional import ?

Reply:
no, the `driver_executable = compute_driver_executable()` into the else branch. Since we don't need it when the if is true, only when its false.